cookbooks/plume_2D_chunk/doc/plume.tex

```diff
@@ -0,0 +1,98 @@
+
+The viscosity expression can be written as
+\[
+\eta(T)  
+= \frac12 \underbrace{2 \eta_0  \exp\left( -\frac{E}{R T\textsubscript{patch}} \right) }_{A^{-1}} \exp \frac{E}{R T}  
+\]
+which is effectively a diffusion creep-type viscosity.
+We find that $A^{-1} = \{ 2.5\times 10^{22}, 1.98\times 10^{22} , 1.6\times 10^{21} \}\si{\pascal\second}$, or  $A = \{ 4\times 10^{-24}, 5.05\times 10^{-24},  6.26\times 10^{-23}\} $. As in Kellogg \& King (1997) the viscosity is limited to  $\eta\textsubscript{max}=1000\eta_0$. 
+
+We run the model to steady state for the reason given in Redmond \& King \cite{reki04} (which is similar to Kellogg \& King \cite{keki97}, only applied to Mars): ``We use steady state calculations so that we can separate time-dependent from parameter-dependent effects. Once again, it is unlikely that any Mars-sized, or larger, planetary body is in steady state. These calculations mainly serve as a guide, allowing us to determine the relationship between surface observations and internal parameters.''
+
+Steady-state fields are shown in Fig.~\ref{fig:plume-diff-creep}. We find that the velocity fields are similar between the isoviscous and weakly temperature-dependent cases, where the convection cell occupies most of the domain. In contrast, the strongly temperature-dependent experiment showcases a large viscosity zone a few hundreds of km's thick (the thermal lithosphere), forcing the convection cell below it.i In other words, the thick lid insulates the convection cell, raising the temperature within it.
+
+\begin{figure}
+  \centering
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/vel1}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/vel2}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/vel3}\\
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/T1}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/T2}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/T3}\\
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/eta1}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/eta2}
+  \includegraphics[width=5cm]{cookbooks/plume_2D_chunk/doc/eta3}
+  \caption{\it Plume in a 2D chunk: Columns from left to right: isoviscous case, weakly temperature dependent case, and strongly temperature-dependent case. Rows from top to bottom: Velocity, temperature, and viscosity field at steady state. Angular opening of $\pi/8$.}
+  \label{fig:plume-diff-creep}
+\end{figure}
+
+
+The obtention of a steady state is contingent on the narrow angular opening. We find that simply increasing the angular opening from $\pi/8$ to $\pi/4$ yields only a statistical steady state, as multiple downwellings occur near the side but the system never stabilises (see Fig.~\ref{fig:plume-angular-opening}). Also, decreasing $\eta_0$ by a factor 10 would yield $Ra=10^7$. In this case, too, a statistical steady state is reached (not shown here).
```

Comment:
Is "obtention" a word? Maybe say "Obtaining a steady state..."

Reply:
obtention is a legit word, but I like your alternative phrasing. Changed it.